frontend/src/Editor/CodeBuilder/TypeMapping.js

```diff
@@ -9,4 +9,5 @@ export const TypeMapping = {
   alignButtons: 'AlignButtons',
   number: 'Number',
   boxShadow: 'BoxShadow',
+  clientServerSwitch: 'ClientServerSwitch',
```

Comment:
@kavinvenkatachalam We could have, but the since we have a particular logic on onChange to either select client side or server side depending on boolean value, I feel the tab component wouldn't have been generic in a way we want it to be.

Reply:
@nakulnagargade In my opinion it should be generic and all the logic should be handled outside this component.